app/concepts/todo/create.rb

```diff
@@ -0,0 +1,9 @@
+class Todo::Create < Trailblazer::Operation
+	def process(params)
+		title = params[:title] || ''
+		description = params[:description] || ''
+		user = User.find(params[:user_id])
+		@valid = user.todos.create({title: title, 
+			description: description})
+	end
+end
```

Comment:
Hi Raymond, from somebody who have no prior trailblazer knowledge, good job! As you might already know, what you accomplished here is a good step towards business logic code reuse. Now, both controllers and specs can avoid duplication by using this single class when creating a todo item. Can you do some improvements? Here's some hints:

`Operations` have a standard way of setting/loading user inputs i.e. `title`, `description`, `user_id` in this case. Try to make use of the `contract do...end` block inside this as well as your other operations.

This is a great start. I'm pretty sure you'll be able to do it because according to Sam and Julian, you're amazingly a fast learner.